### PR TITLE
New navigation + updated content

### PR DESCRIPTION
**Pull request by allisonnorman**
- Adds navigation and sub-navigation for each of the sections
- Reorganizes the content markdown pages and folders into a _pages folder
- Adds new content from draft document

### CODE REVIEW

File: _pages/define/problem.md [hunk 1]
```diff
@@ -1,51 +1,56 @@
...
+sidenav: define
+sticky_sidenav: true
 ---
 
-It's critical that you deeply understand the problem you're solving. This means that once you understand the problem, you are comfortable exploring and pivoting to different solutions. 
+It's critical that you deeply understand the problem you're solving. This means that once you understand the problem, you are comfortable exploring, evaluating, and pivoting to different solutions.
 
 Just as importantly, you need to bring everyone along with you. Your partners and teammates need to agree on the problem you're tackling and the problems you are not tackling. A well-written problem statement helps align and motivate your team around a common goal, providing focus and keeping distractions at bay.
 
-### Considerations: A good problem statement
```
Comment (brentryanjohnson) [inline _pages/define/problem.md]: This page (perhaps others, too) jumps from an `h1` to `h3`. For accessibility, the headings should precede sequentially from `h1` to `h2` to `h3` and so on, with `h3`s as "children" of `h2`s.

File: _pages/define/vision.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+sidenav: define
+sticky_sidenav: true
+---
+
+A vision statement acts as a north star that motivates and guides the team, conveying the overarching goal the team is aiming for. It lets everyone tell the same story, with the same passion and conviction. It’s also a short “elevator pitch” that helps you communicate clearly about the value of the work you’re doing to anyone at any time so you can quickly galvanize support.
+
+While you can undoubtedly come up with a great vision for the project all by yourself, it’s not sustainable—because you’re going to walk away from this project and leave it in your partner’s hands. Our goal is to help our partners grow their internal practices, and teaching them how to set a strong vision is an important step on that journey. You will revisit and may adjust your vision at times as you learn more, but maintaining a clear, inclusive vision is critical to product success.
+
+
+### Considerations: A good vision statement...
```
Comment (brentryanjohnson) [inline _pages/define/vision.md]: See previous comment about heading levels.

File: _data/navigation.yml [hunk 3]
```diff
@@ -1,43 +1,61 @@
...
+    - text: Discover the current state
+      href: /discover/
+    - text: Understand users and their 'as is' process
+      href: /discover/users/
+    - text: Understand your stakeholders
+      href: /discover/stakeholders/
+    - text: Understand the technical landscape
+      href: /discover/technical/
+    - text: Understand the regulatory landscape
+      href: /discover/compliance/
```
Comment (brentryanjohnson) [inline _data/navigation.yml]: It looks like there's some variability in the indentation of the `yaml` list items. It doesn't appear to be breaking anything, but it's probably a good idea to use consistent spacing. But don't tell anyone whether you're using tab versus spaces ;)

(I recommend 2 spaces, but don't tell anyone.)

File: _pages/define/problem.md [hunk 4]
```diff
@@ -1,51 +1,56 @@
...
--   Use the problem statement to create the vision
+- Use the problem statement to create the vision.
 
--   Use the problem statement to surface and resolve possible disagreements among stakeholders
+- Use the problem statement to surface and resolve possible disagreements among stakeholders.
 
--   As you absorb new information, regularly reconsider the problem and update as appropriate
+- As you absorb new information, regularly reconsider the problem and update as appropriate.
 
-### RESOURCES
```
Comment (brentryanjohnson) [inline _pages/define/problem.md]: Avoid all caps for headings. 18F content guide recommends sentence case.

**Review by brentryanjohnson — CHANGES_REQUESTED** [review]
I added some comments and suggestions. Looks good overall, and it seems to be building fine!

File: _data/navigation.yml [hunk 5]
```diff
@@ -1,43 +1,61 @@
...
-        href: /discover/technical/
-      - text: Understand the regulatory landscape
-        href: /discover/compliance/
+    href: /discover/
   - text: Define the future state
-    links:
-      - text: Overview
-        href: /future/
+    href: /define/
+  - text: Deliver the "right" way
```
Comment (brentryanjohnson) [inline _data/navigation.yml]: You might want to replace these with smart quotes, both for aesthetic and programmatic reasons. `option + [` for opening and `option + shift + [` for close should do it for you.

File: _data/navigation.yml (hunk 3 above)
Comment (brentryanjohnson) [reply _data/navigation.yml]: (I'm pretty sure `YAML` doesn't impose strict indenting rules, but it's good to be consistent for readability and consistency.)

Comment (allisonnorman) [reply _data/navigation.yml]: Good catch! I thought my prettier plugin would have caught that but just noticed it was disabled. Thanks!

File: index.md [hunk 6]
```diff
@@ -1,11 +1,14 @@
...
 
-**You will also focus on sustainability.** Anything we build should empower our partner agencies to better meet their mission, be relatively easy for them to adopt and support, and be easy for end users to use. If we are building something with a partner, the product needs to be constructed and hosted in a way that the partner can manage, and the partner should be empowered to take over after we leave. We'll talk more about this in the [Enable partners for long-term success]({{ "/partners/enable/" | relative_url }}) section.
\ No newline at end of file
+**You will also focus on sustainability.** Anything we build should empower our partner agencies to better meet their mission, be relatively easy for them to adopt and support, and be easy for end users to use. If we are building something with a partner, the product needs to be constructed and hosted in a way that the partner can manage, and the partner should be empowered to take over after we leave. We'll talk more about this in the [Enable partners for long-term success]({{site.baseurl}}/partners/) section.
```
Comment (brentryanjohnson) [inline index.md:14]: I think the `relative_url` filter [is preferable now](https://ben.balter.com/jekyll-style-guide/config/#url) to calling `site.baseurl`, but if it's working like this, it probably doesn't matter that much.

File: _data/navigation.yml (hunk 5 above)
Comment (allisonnorman) [reply _data/navigation.yml]: 👍

**Review by allisonnorman — COMMENTED** [review]
Changes made! Thanks @brentryanjohnson! 

Specifically:
- h2 continuity
- yaml formatting
- smart quotes added

File: _data/navigation.yml [hunk 7]
```diff
@@ -11,51 +11,51 @@ primary:
...
 deliver:
-    - text: Deliver the "right" way
-      href: /deliver/
-    - text: Build the product
-      href: /deliver/build/
-    - text: Collaborate and communicate
-      href: /deliver/collaborate/
-    - text: Measure success
-      href: /deliver/measure/
+  - text: Deliver the "right" way
```
Comment (brentryanjohnson) [inline _data/navigation.yml]: @allisonnorman You might be able to use smart quotes here, too.

**Review by brentryanjohnson — APPROVED** [review]

File: _data/navigation.yml (hunk 7 above)
Comment (brentryanjohnson) [reply _data/navigation.yml]: I took care of this one, @allisonnorman.

**Pull request merged**